client/app/scripts/superdesk/superdesk.js

```diff
@@ -97,8 +97,13 @@ define([
         }]);
 
     // make sure there is templates module defined
-    angular.module('templates', []);
-    modules.push('templates');
+    try {
```

Comment:
It is there because the previous template module was added here before. (Look at the removed lines)
And this is the template file of the superdesk module only, isn't it @thnkloud9  ?